src/resources/postcss/components/_view-selector.pcss

```diff
@@ -0,0 +1,273 @@
+		}
+	}
+
+	.tribe-events-c-view-selector--tabs {
+		@media (--viewport-medium) {
+			width: auto;
+		}
+
+		.tribe-events-c-view-selector__button {
+			@media (--viewport-medium) {
+				@mixin hidden;
+			}
+		}
+
+		.tribe-events-c-view-selector__content {
+			@media (--viewport-medium) {
+				border: none;
+				border-radius: 0;
+				box-shadow: none;
+				display: block;
+				padding: 0 calc(var(--spacer-4) / 2);
+				position: static;
+				transform: none;
+			}
+		}
+
+		.tribe-events-c-view-selector__list {
+			@media (--viewport-medium) {
+				display: flex;
+			}
+		}
+
+		.tribe-events-c-view-selector__list-item {
+			@media (--viewport-medium) {
+				margin: 0 calc(var(--spacer-4) / 2);
+			}
+		}
+
+		.tribe-events-c-view-selector__list-item-link {
+			@media (--viewport-medium) {
+				padding: var(--spacer-3) 0;
+				position: relative;
+			}
+		}
+
+		.tribe-events-c-view-selector__list-item-link--active {
+			@media (--viewport-medium) {
+				background-color: transparent;
+			}
+
+			&:after {
+				@media (--viewport-medium) {
+					background-color: var(--color-accent-secondary);
+					bottom: -1px;
+					content: '';
+					display: block;
+					height: 2px;
+					position: absolute;
+					width: 100%;
+				}
+			}
+		}
+	}
+
+	.tribe-events-c-view-selector__button {
+		align-items: center;
+		display: flex;
+		padding: var(--spacer-0);
+		position: relative;
+		text-align: left;
+		width: 100%;
+
+		@media (--viewport-medium) {
+			padding: var(--spacer-3) var(--spacer-4);
+		}
+
+		&:hover,
+		&:focus {
+			border-bottom-color: var(--color-accent-primary);
+
+			&:before {
+				opacity: 1;
+			}
+		}
+
+		&:before {
+			background-color: var(--color-accent-primary);
+			content: '';
+			display: block;
+			height: 2px;
+			opacity: 0;
+			position: absolute;
+			top: 100%;
+			transform: translateY(var(--spacer-0));
+			width: calc(100% - (2 * var(--spacer-0)));
+
+			@media (--viewport-medium) {
+				content: none;
+			}
+		}
+
+		&:after {
+			@media (--viewport-medium) {
+				background-image: svg-inline(caret-down);
+				background-repeat: no-repeat;
+				background-size: contain;
+				content: '';
+				display: block;
+				flex: none;
+				height: 6px;
+				margin-left: var(--spacer-1);
+				width: 10px;
+			}
+		}
+	}
+
+	.tribe-events-c-view-selector__button--active {
+
+		&:after {
+			@media (--viewport-medium) {
+				background-image: svg-inline(caret-up);
+			}
+		}
+	}
+
+	.tribe-events-c-view-selector__button-icon {
+		content: '';
+		display: block;
+		height: 21px;
+		width: 21px;
+
+		@media (--viewport-medium) {
+			@mixin hidden;
+		}
+	}
+
+	.tribe-events-c-view-selector__button-text {
+		@mixin visually-hide;
+
+		@media (--viewport-medium) {
+			@mixin visually-show;
+			@mixin desktop-body-2-bold;
+
+			flex: auto;
+		}
+	}
+
+	.tribe-events-c-view-selector__content {
+		display: none;
+		background-color: var(--color-background);
+		border: 1px solid var(--color-border-secondary);
+		border-radius: 4px;
```

Comment:
We're repeating the same border radius on buttons on some other places. Maybe it's a good time to define this as a variable and use it on other places, as it looks like it'll be part of the design system.